src/devtools/client/inspector/markup/new-markup.js

```diff
@@ -219,13 +241,13 @@ class MarkupView {
     // starting from the selected node itself since we want to display it expanded with
     // its children visible.
     while (currentNode) {
-      if (this.nodes.has(currentNode.objectId())) {
+      if (!this.nodes || this.nodes.has(currentNode.objectId())) {
```

Comment:
Probably can simplify to use optional chaining with `this.nodes?`

Reply:
No, `this.nodes?.has(...)` would return `undefined` (which is falsy) if `this.nodes` is undefined, but we need it to return `true` (or something truthy) in that case.